osu.Game/Rulesets/Mods/ModFlashlight.cs

```diff
@@ -27,21 +28,47 @@ namespace osu.Game.Rulesets.Mods
 {
     public abstract class ModFlashlight : Mod
     {
+        private const int min_change_size_combo = 1;
```

Comment:
What does 1 even mean here?

Reply:
not any particular reason is just that i though this should be the default for both the settings, if you want me to move this from a constant, sure?

Reply:
This is not used anymore.